app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  FIXTURE_DIR = File.join(Rails.root, "db/fixtures")
```

Comment:
Prefer `Rails.root.join("db/fixtures")`.  This way it's also a Pathname for further joining.